runtime/compiler/runtime/IProfiler.cpp

```diff
@@ -4774,7 +4774,7 @@ void TR_AggregationHT::sortByNameAndPrint(TR_J9VMBase *fe)
             fprintf(stderr, "Cannot allocate memory. Incomplete data will be printed.\n");
             break;
             }
-         sprintf(wholeName, "%.*s.%.*s%.*s",
+         snprintf(wholeName, len, "%.*s.%.*s%.*s",
```

Comment:
`len` should be one more (+3 for two `.` and one `NUL`).

Reply:
There is only one `.` in the pattern.

Reply:
Sorry, you're right.